### PR TITLE
Event onclick

### LINKED ISSUE #81: Clicked Component Functunality
As a user when I click a week I want to know what of my events  occurred on that week.

### PR DESCRIPTION
**Pull request by taylorjohnson141**
### Code Highlights:

Adds ability for user to click an event and see what happened
 
### Where should the reviewer start?

 clickedComponent.tsx

### Have Tests Been Added?
- [x] No.
- [ ] Yes, but all tests are not passing.
- [ ] Yes, and all are passing.

### Any background context you want to provide?

Before user would get nothing happened here message if they clicked on an event week.

### Issues:
* Addresses #
* Closes #81 

### Tracking Consistency:
- [x] added appropriate labels
- [x] My code follows the code style of this project and has removed all unnecessary annotations
- [x] I have added comments on my pull request, particularly in hard-to-understand areas
- [x] looked at PR preview to check spelling, syntax, formatting, and completion

### CODE REVIEW

File: src/Components/CalendarComponent/CalendarComponent.tsx [hunk 1]
```diff
@@ -123,6 +123,7 @@ const CalendarComponent = (props: CalenderComponentProps) => {
       if (currentEvent) {
         return (
           <Week
+           startDate = {currentEvent.date}
```
Comment (taylorjohnson141) [inline src/Components/CalendarComponent/CalendarComponent.tsx:126]: Before this was not being passed as prop causing the nothing happened here message to display.

File: src/Components/ClickedComponent/ClickedComponent.tsx [hunk 2]
```diff
@@ -19,10 +19,20 @@ let ClickedComponent = (props: ClickedComponentProps) =>{
       return newArr.join('/')
     }
     let startDate = updateDate(props.startDate)
-    let endDate = updateDate(props.endDate)
+    let endDate;
+    if(props.endDate){
```
Comment (taylorjohnson141) [inline src/Components/ClickedComponent/ClickedComponent.tsx:23]: Updated logic to display correct message depending on if there is an end event or not

File: src/Components/CalendarComponent/CalendarComponent.tsx (hunk 1 above)
Comment (krogowsk531) [reply src/Components/CalendarComponent/CalendarComponent.tsx:126]: Good catch!

File: src/Components/ClickedComponent/ClickedComponent.tsx (hunk 2 above)
Comment (krogowsk531) [reply src/Components/ClickedComponent/ClickedComponent.tsx:23]: Awesome!

**Review by krogowsk531 — APPROVED** [review]
Thanks for getting this feature working properly!

File: src/Components/CalendarComponent/CalendarComponent.tsx (hunk 1 above)
Comment (aemcdonald) [reply src/Components/CalendarComponent/CalendarComponent.tsx:126]: Awesome catch, makes sense!

**Review by aemcdonald — APPROVED** [review]
Thank you for working on this! Great job accounting for a single date for an event or a range for an era.

**Pull request merged**